corpora/proverbs.dep

```diff
@@ -27,3 +28,979 @@
+"<сүйлөсүн>"
+	"сүйлө" v tv opt p3 sg @root #8->0
+"<.>"
+	"." sent @punct #9->8
+
+# sent_id = proverbs:16
+# author:id = Сыргабековна&Сабыровна:84
+# text = Балалуу болбогон кубанганды билбеген.
+# text[eng] = Those who have had children have not known being happy.
+# label_cg = JNW_checked
+# label_dep = revised_1time
+"<Балалуу>"
+	"бала" n @xcomp #1->3
+		"луу" post @case #2->1
+"<болбогон>"
+	"бол" v iv gpr_past subst nom @nsubj #3->5
+"<кубанганды>"
+	"кубан" v iv ger_past acc @obj #4->5
+"<билбеген>"
+	"бил" v tv neg past p3 sg @root #5->0
+"<.>"
+	"." sent @punct #6->5
+
+# sent_id = proverbs:17
+# author:id = Сыргабековна&Сабыровна:6
+# text = Адамдын көркү - чүпүрөк, жыгачтын көркү - жалбырак.
+# text[eng] = The elegance of a person is clothes, the elegance of a tree is leaves.
+# label_cg = JNW_checked
+# label_dep = revised_1time
+"<Адам>"
+	"адам" n gen @nmod:poss #1->2
+"<көркү>"
+	"көрк" n px3sp nom @nsubj #2->4
+"<->"
+	"-" guio @punct #3->7
+"<чүпүрөк>"
+	"чүпүрөк," n nom @parataxis #4->3
+"<,>"
+	"," cm @punct #5->9
+"<жыгачтын>"
+	"жыгач" n gen @nmod:poss #6->7
+"<көркү>"
+	"көрк" n px3sp nom @nsubj #7->9
+"<->"
+	"-" guio @punct #8->9
+"<жалбырак>"
+	"жалбырак" n nom @root #9->0
+		"э" cop aor p3 sg @cop #10->9
+"<.>"
+	"." sent @punct #11->9
+
+# sent_id = proverbs:18
+# author:id = Сыргабековна&Сабыровна:298
+# text = Көп жашасаң, көптү көрөсүң.
+# text[eng] = If you live a lot (of time), you will see a lot.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Көп>"
+	"көп" adv @advmod #1->2
+"<жашасаң>"
+	"жаша," v iv gna_cond p2 sg @advcl #2->5
+"<,>"
+	"," cm @punct #3->5
+"<көптү>"
+	"көп" adj subst acc @obj #4->5
+"<көрөсүң>"
+	"көр" v tv aor p2 sg @root #5->0
+"<.>"
+	"." sent @punct #6->5
+
+# sent_id = proverbs:19
+# author:id = Сыргабековна&Сабыровна:348
+# text = Оюндан от чыгат.
+# text[eng] = From playing comes fire.
+# label_cg = JNW_checked
+# label_dep = revised_1time
+"<Оюндан>"
+	"оюн" n abl @obl #1->3
+"<от>"
+	"от" n nom @nsubj #2->3
+"<чыгат>"
+	"чык" v iv aor p3 sg @root #3->0
+"<.>"
+	"." sent @punct #4->3
+
+# sent_id = proverbs:20
+# author:id = Сыргабековна&Сабыровна:249
+# text = Кайра кирер эшигиңди катуу жаппа.
+# text[eng] = Don't close a door firmly that you might go back through. (="Don't burn bridges.")
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Кайра>"
+	"кайыр" v tv gna_impf @advmod #1->5
+"<кирер>"
+	"кир" v iv gpr_fut @amod #2->3
+"<эшигиңди>"
+	"эшик" n px2sg acc @obj #3->5
+"<катуу>"
+	"катуу" adv @advmod #4->5
+"<жаппа>"
+	"жап" v tv neg imp p2 sg @root #5->0
+"<.>"
+	"." sent @punct #6->5
+
+# sent_id = proverbs:21
+# author:id = Сыргабековна&Сабыровна:283
+# text = Колуң менен кылганды, мойнуң менен тартасың.
+# text[eng] = You pull with your neck what you make with your hands.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Колуң>"
+	"кол" n px2sg nom @obl #1->3
+"<менен>"
+	"менен" post @cc #2->1
+"<кылганды>"
+	"кыл" v tv gpr_past subst acc @conj #3->7
```

Comment:
This is a clausal object.  I forget what the deprel is for it, but `@conj` is definitely not right.

Reply:
I am not sure either... xcomp?

Reply:
Yes, I think `@xcomp` is right; `@ccomp` doesn't work because it doesn't have a separate subject from the main verb, although really it's hard to tell in Turkic languages because subjects can be "elided" (and in imperatives even in English!)...

Reply:
Ok! I think I changed it to xcomp.

Reply:
Now I'm going back and forth between `@ccomp` and `@obj` here.  It's definitely an object, the question is whether it's clausal or or just an elided noun (that is in turn modified by the `gpr`).  This is an interesting question and needs further discussion!